lib/client/keystore.go

```diff
@@ -267,9 +277,12 @@ func (fs *FSKeyStore) DeleteKey(idx KeyIndex) error {
 // Useful when needing to log out of a specific service, like a particular
 // database proxy.
 func (fs *FSKeyStore) DeleteUserCerts(idx KeyIndex, opts ...CertOption) error {
```

Comment:
nit: the godoc is not up-to-date anymore as it mentions the function only cleans-up the certs associated with the user private key. If I understand correctly this also deletes the individual app keypair now.